PocketKit/Sources/Analytics/AppEvents/Search.swift

```diff
@@ -45,21 +47,33 @@ public extension Events.Search {
         positionInList: Int,
         scope: SearchScope
     ) -> Event {
-        var identifier = ""
-        switch scope {
-        case .saves:
-            identifier = "global-nav.search.saves.unfavorite"
-        case .archive:
-            identifier = "global-nav.search.archive.unfavorite"
-        case .all:
-            identifier = "global-nav.search.all.unfavorite"
-        }
+        return Engagement(
+            .general,
+            uiEntity: UiEntity(
+                .button,
+                identifier: "global-nav.search.unfavorite",
+                componentDetail: getScopeIdentifier(scope: scope),
```

Comment:
conversation on analytics ongoing here: https://pocket.slack.com/archives/C01UH57EJKD/p1676996499616879

will refine analytics in a different PR